test/Tests/TestPartyCreationMenu.cpp

```diff
@@ -12,20 +12,22 @@ GAME_TEST(Items, GenerateItem) {
         pItemTable->GenerateItem(ITEM_TREASURE_LEVEL_6, 0, &item);
 }
 
-GAME_TEST(Menu, PartyCreation) {
-    EXPECT_EQ(GetCurrentMenuID(), MENU_MAIN);
-
-    game->pressGuiButton("MainMenu_NewGame");
-    game->tick(2);
-
-    EXPECT_EQ(GetCurrentMenuID(), MENU_CREATEPARTY);
-    EXPECT_EQ(pParty->pPlayers[0].uMight, 30);
-
-    game->pressGuiButton("PartyCreation_Clear"); // This shouldn't crash.
-    game->tick();
-
-    EXPECT_EQ(pParty->pPlayers[0].classType, PLAYER_CLASS_KNIGHT);
-    EXPECT_EQ(pParty->pPlayers[0].uMight, 14);
+GAME_TEST(Prs, Pr314) {
```

Comment:
lets call  it something more meaningful..

Reply:
The naming that's currently used is:
IssueXXX for tests that check existing issues.
PrXXX for issues that were introduced / fixed in a PR and for which an issue was never created.

I guess we can use PrXXX_WhatWentWrong and IssueXXX_WhatDoWeTestHere instead. What do you think?

Reply:
Naming is fine as is with a small comment in each block saying what it's testing for.